docs/docusaurus.config.js

```diff
@@ -72,9 +73,6 @@ const config = {
         docs: {
           path: "docs",
           routeBasePath: "docs",
-          admonitions: {
-            icons: "emoji",
-          },
```

Comment:
This throws an error with the new docusaurus version. Admittedly I dont know exactly what this does or how it works, so I could use some help here.

Reply:
This is the error:
```
[ERROR] Error: Unable to build website for locale en.
    at tryToBuildLocale (/Users/kcp/projects/nebari_project/nebari-docs2/docs/node_modules/@docusaurus/core/lib/commands/build.js:54:19)
    at async /Users/kcp/projects/nebari_project/nebari-docs2/docs/node_modules/@docusaurus/core/lib/commands/build.js:65:9
    at async mapAsyncSequential (/Users/kcp/projects/nebari_project/nebari-docs2/docs/node_modules/@docusaurus/utils/lib/jsUtils.js:21:24)
    at async Command.build (/Users/kcp/projects/nebari_project/nebari-docs2/docs/node_modules/@docusaurus/core/lib/commands/build.js:63:5) {
  [cause]: [Error [ValidationError]: "admonitions.icons" is not allowed] {
    _original: {
      path: 'docs',
      routeBasePath: 'docs',
      admonitions: { icons: 'emoji' },
      sidebarPath: '/Users/kcp/projects/nebari_project/nebari-docs2/docs/sidebars.js',
      sidebarCollapsible: true,
      editUrl: 'https://github.com/nebari-dev/nebari-docs/tree/main/docs',
      showLastUpdateAuthor: false,
      showLastUpdateTime: true
    },
    details: [
      {
        message: '"admonitions.icons" is not allowed',
        path: [ 'admonitions', 'icons' ],
        type: 'object.unknown',
        context: {
          child: 'icons',
          label: 'admonitions.icons',
          value: 'emoji',
          key: 'icons'
        }
      }
    ]
  }
}
```

Reply:
My understanding of this is that previously, we had to manually tell docusaurus to render emojis in markdown, and now this happens by default. After I remove these lines, the emojis still render so I believe this is fine to remove as no longer necessary.